internal/distro/rhel9/distro.go

```diff
@@ -156,16 +125,19 @@ func (d *distribution) getDefaultImageConfig() *distro.ImageConfig {
 }
 
 func New() distro.Distro {
-	return NewRHEL90()
+	// default minor: create default minor version (current GA) and rename it
+	d := newDistro("rhel", 0)
```

Comment:
Same here, current GA is 9.1. Sorry, we are pushing these two fast. :D

Reply:
This is fixed now.